package.json

```diff
@@ -17,30 +17,41 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-vc-verifier",
   "dependencies": {
-    "axios": "^0.19.2",
-    "bedrock-did-context": "^1.0.0",
-    "bedrock-veres-one-context": "^10.0.0",
+    "@digitalbazaar/ed25519-signature-2018": "^2.0.1",
+    "@digitalbazaar/ed25519-signature-2020": "^2.2.0",
+    "@digitalbazaar/ed25519-verification-key-2018": "^3.1.1",
+    "@digitalbazaar/http-client": "^1.1.0",
+    "@digitalbazaar/vc": "^1.0.0",
+    "bedrock-did-context": "^2.0.0",
+    "bedrock-security-context": "^4.1.0",
+    "bedrock-veres-one-context": "^11.0.0",
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
-    "did-io": "^0.8.2",
-    "did-method-key": "^0.6.1",
-    "did-veres-one": "^12.1.1",
-    "serialize-error": "^6.0.0",
-    "vc-js": "^0.6.1",
-    "vc-revocation-list": "^1.0.0"
+    "crypto-ld": "^6.0.0",
+    "did-context": "^3.0.1",
+    "serialize-error": "^8.1.0",
+    "vc-revocation-list": "^1.0.0",
+    "veres-one-context": "^12.0.0"
   },
   "peerDependencies": {
-    "bedrock": "^3.1.0",
+    "bedrock": "^4.1.1",
     "bedrock-credentials-context": "^1.0.0",
-    "bedrock-express": "^3.1.0",
-    "bedrock-https-agent": "^1.0.1",
+    "bedrock-did-io": "digitalbazaar/bedrock-did-io#support-2018",
```

Comment:
Still waiting on this?  I don't see a PR https://github.com/digitalbazaar/bedrock-did-io/tree/support-2018

Reply:
Here's the PR - https://github.com/digitalbazaar/bedrock-did-io/pull/4 (it was like a one-line change, I was waiting on the did-veres-one PR to go through, before publishing this one)